lib/services/elasticsearch.js

```diff
@@ -251,11 +277,51 @@ function ElasticSearch (kuzzle, options, config) {
     var
       data = cleanData.call(this, requestObject);
 
-    // extends the response with the source from requestObject
-    // When we write in ES, the response from it doesn't contain the initial document content
+    // Add metadata
+    data.body._kuzzle_info = {
+      author: requestObject.userId ? String(requestObject.userId) : null,
+      createdAt: new Date(),
```

Comment:
```new Date()``` will return a date formatted with local time (ex: ```Mon Oct 17 2016 14:34:29 GMT+0000 (UTC)```) 

I'll suggest to use ```Date.now()``` function instead.
Or define mapping to recognize a specific date format and format date field with momentjs: https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-date-format.html 
http://momentjs.com/docs/#/displaying/format/